analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/OperationProcessors/Invocation.cs

```diff
@@ -72,6 +120,24 @@ protected override ProgramState[] Process(SymbolicContext context, IInvocationOp
         };
     }
 
+    private static IOperation CanLearnNotNull(IInvocationOperationWrapper invocation)
```

Comment:
What about this? I'd expect `bool` return from a `Can` method
```suggestion
    private static IOperation LearnNotNullCandidate(IInvocationOperationWrapper invocation)
```